### DISCUSSION TITLE
Week 1 Assignment

### DISCUSSION THREAD

**Discussion by sraorao** [sraorao/MSD_R_course_MT2021#3] [Show and tell]
Q1. Packages can be installed in R from various repositories. The most common ones are CRAN and Bioconductor. Install the beeswarm package from CRAN.

Q2. The beeswarm package comes with a preloaded dataset of breast tumours. Load this dataset into your global R environment. hint: You first need to load the package into R and then load the breast data.frame into memory; try ??breast at your R console and follow the help pages if needed.

Q3. How many ER positive and ER negative tumours are in the dataset?

Q4. Plot ESR1 on the X axis and ERBB2 on the y axis, colour the dots by ER status

Q5. Filter the dataset to include only those patients who have metastasis as the outcome, and assign this filtered dataset to a new variable.

**Comment by sammurray13** [comment]
ER POS = 209
ER NEG = 77

> with(breast,
+      plot(col = as.numeric(ER), ERBB2))

> metast <- breast[event_survival==1]
